docs/developer/payments.mdx

```diff
@@ -8,12 +8,301 @@ The process below describes the key milestones in the Saleor payment flow.
 Additional steps may also occur along the way; however, the purpose of this instruction is to
 deliver a base reference for the user to work with.
 
-Saleor distinguishes two different ways of processing a payment:
+Saleor distinguishes three different ways of processing a payment:
 
 - By using a [plugin](extending/overview#plugins) embedded in the Saleor code.
-- By using a [Saleor App](extending/overview#apps-recommended) (recommended).
+- By using a [Saleor Payment App](extending/overview#apps-recommended)
+- By using a [Saleor App with existing Checkout App](extending/overview#apps-recommended).
```

Comment:
These two links are the same

Reply:
Yeap. They points to general concept of the `App`. Do you think that I should link something different?

Reply:
Nevermind. I already changed this structure